src/main/kotlin/com/sourcegraph/cody/chat/actions/BaseCommandAction.kt

```diff
@@ -0,0 +1,27 @@
+package com.sourcegraph.cody.chat.actions
+
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.openapi.fileEditor.FileEditorManager
+import com.intellij.openapi.project.Project
+import com.sourcegraph.cody.CodyToolWindowContent
+import com.sourcegraph.cody.autocomplete.CodyEditorFactoryListener
+import com.sourcegraph.cody.chat.AgentChatSession
+import com.sourcegraph.cody.commands.CommandId
+
+abstract class BaseCommandAction : BaseChatAction() {
+
+  abstract fun myCommandId(): CommandId
```

Comment:
We can use `abstract val myCommandId: CommandId` here, no need to define this as function

Reply:
I will fix that in other PR